packages/react-native/ReactCommon/yoga/yoga/YGEnums.h

```diff
@@ -38,7 +38,8 @@ YG_ENUM_DECL(
 YG_ENUM_DECL(
     YGDisplay,
     YGDisplayFlex,
-    YGDisplayNone)
+    YGDisplayNone,
+    YGDisplayContents)
```

Comment:
> I'm not exactly sure what's the best way to fully separate Display::Contents from Yoga since the two places where display property is stored are the yoga node and props which hold yoga styles

The Props structure holding a `yoga::Style` is something I plan to change in the future more broadly. These should ideally not be coupled together.

We have some extra props on YogaStylableProps not part of `yoga::Style` that we flatten when transferring props to Yoga node. Maybe we could back display prop by field, and ignore the one on the Yoga style for now during props part?

Reply:
Is [this](https://github.com/facebook/react-native/pull/46584/commits/e9f41401afc624c156e668132707f7aac7ab76ae) close to what you had in mind?